sdk/ai/Azure.AI.Agents/src/Azure.AI.Agents.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>
+      The official .NET library for Azure AI Foundry Agents, powered by the OpenAI Responses API.
+      The Azure AI Foundry Agents service includes capabilities to create, manage, and use a variety of declarative, code-based, and workflow agent definitions as part of an agentic application workflow.
+    </Description>
+    <AssemblyTitle>Azure SDK Code Generation Azure.AI.Agents for Azure Data Plane</AssemblyTitle>
+    <Version>2.0.0</Version>
+    <PackageTags>Azure.AI.Agents</PackageTags>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <NoWarn>$(NoWarn);OPENAI001;CS1591;AZC0007;AZC0012;AZC0015;AZC0030;AZC0031;AZC0034;SA1649;SA1402;</NoWarn>
+    <IncludeOperationsSharedSource>true</IncludeOperationsSharedSource>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="$(AzureCoreSharedSources)AzureResourceProviderNamespaceAttribute.cs" LinkBase="Shared/Core" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.Core" />
+    <PackageReference Include="System.Text.Json" />
```

Comment:
```suggestion
```

We get this transitively from Core and shouldn't use a direct reference unless there's a need to hoist the version locally.